api/apps/restful_apis/chunk_api.py

```
@@ -704,6 +708,61 @@ def _group_ids(raw) -> list[str]:
+            index_name,
+            [dataset_id],
+        )
+        raptor_rows = settings.docStoreConn.get_fields(res_raptor, ["content_with_weight", "compile_kwd"]) or {}
+    except Exception:
+        logging.exception("structure graph: RAPTOR blob load failed for doc=%s", document_id)
+        raptor_rows = {}
+    for row in raptor_rows.values():
+        try:
+            graph = json.loads(row.get("content_with_weight") or "{}")
+        except Exception:
+            continue
```

Ruff (0.16.3)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)